x-pack/plugins/security_solution/public/flyout/README.md

```diff
@@ -1,44 +1,61 @@
-# expandable flyout panels
+# Security Solution expandable flyouts
+
+For more info on the expandable flyout, see the `@kbn/expandable-flyout` package.
 
 ## Description
 
-This folder hosts the panels that are displayed in the expandable flyout (see `@kbn/expandable-flyout` package).
+The Security Solution plugin aims at having a single instance of the expandable flyout. That instance can display as many panels as we need. This folder hosts all the panels that are can be displayed in the Security Solution flyout. Panels can be differentiated as to be displayed in different sections of the expandable flyout (right, left or preview), but ultimately, nothing prevents us from displaying a panel in any section we want.
 
-> Remember to add any new panels to the `index.tsx` at the root of the `flyout` folder. These are passed to the `@kbn/expandable-flyout` package as `registeredPanels`.
+> Remember to add any new panels to the `index.tsx` at the root of the [flyout folder](https://github.com/elastic/kibana/tree/main/x-pack/plugins/security_solution/public/flyout). These are passed to the `@kbn/expandable-flyout` package as `registeredPanels`. Failing to do so will result in the panel not being rendered.
 
 ## Notes
 
-At the moment, we only have a single expandable flyout for the Security Solution plugin. This flyout will be used for all documents (signals, events, indicators, assets and findings). We're using a set of generic right/left/preview panels, hence the following folder structure:
-
+The structure of the `flyout` folder is intended to work as follows:
+- top level folders refer to the _type_ of flyout (for example alert details, event details, user, host, rule, cases...) and would contain all the panels for that flyout _type_.
```

Comment:
should `alert details, event details` => `document details`?

Reply:
thank you, fixed!